shakedown/dcos/__init__.py

```diff
@@ -30,12 +29,30 @@ def dcos_service_url(service):
     return _gen_url("/service/{}".format(service))
 
 
+def dcos_exhibitor_url():
+    return "http://{}/".format(master_ip())
```

Comment:
Yes, true -- is that a problem functionally (ie in deleting old ZK nodes), or just with the name of the function here (`dcos_exhibitor_url`)?  I'm proxying through AR rather than connecting to the master and performing the HTTP call from there (as the Docker-based job was doing).